internal/common/vote/vote.go

```diff
@@ -0,0 +1,191 @@
+package vote
+
+import (
+	"math/rand"
+	"time"
+)
+
+func Get_votes_for_rule(ruleID string, islandID int) map[int][]int {
```

Comment:
```suggestion
func GetVotesForRule(ruleID string, islandID int) map[int][]int {
```
Golang does not encourage snake case :)